pkg/minikube/cluster/cluster.go

```diff
@@ -300,9 +303,53 @@ func createVirtualboxHost(config MachineConfig) drivers.Driver {
 	return d
 }
 
+func isISONotCachedInMinikubeDir(isoFilePath string) bool {
```

Comment:
In general "positive" boolean functions are easier to understand. Could you change this to:

`isISOCached`